app/src/components/integrations/modals/addIntegrationModal/addIntegrationModal.jsx

```diff
@@ -62,13 +64,24 @@ const AddIntegrationModal = ({ data, initialize, change, handleSubmit }) => {
   const fieldsExtensions = useSelector(uiExtensionIntegrationFormFieldsSelector);
   const dispatch = useDispatch();
   const { formatMessage } = useIntl();
+  const { trackEvent } = useTracking();
   const { onConfirm, customProps, isGlobal } = data;
 
+  const integrationFieldsExtension = fieldsExtensions.find(
+    (ext) => ext.pluginName === data.instanceType,
+  );
+
   const updateMetaData = (newMetaData) => {
     setMetaData({ ...metaData, ...newMetaData });
   };
 
   const onSubmit = (newData) => {
+    if (isGlobal && !customProps.editAuthMode) {
+      trackEvent(
+        PLUGINS_PAGE_EVENTS.clickCreateGlobalIntegration(integrationFieldsExtension.pluginName),
```

Comment:
Please use `data.instanceType` instead of `integrationFieldsExtension.pluginName` as not all plugins support fields as extensions.